src/utils/__test__/ConfigUtils.spec.js

```diff
@@ -91,3 +91,119 @@ describe('getConversionMethod with possible values', () => {
     }
   );
 });
+
+describe('Check createNewTableLine returns value', () => {
+  const COLUMNS = [
+    // Basic webapp handler DONE
+    {
+      field: 'string',
+    },
+    {
+      field: 'integer',
+      type: ['int'],
+    },
+    {
+      field: 'boolean',
+      type: ['bool'],
+    },
+    {
+      field: 'enum',
+      type: ['enum'],
+    },
+    {
+      field: 'enumWithValues',
+      type: ['enum'],
+      cellEditorParams: { enumValues: ['First', 'Second', 'Third'] },
+    },
+    {
+      field: 'date',
+      type: ['date'],
+    },
+    {
+      field: 'number',
+      type: ['number'],
+    },
+    // Set DefaultValues
+    {
+      field: 'defaultValueString',
+      cellEditorParams: { defaultValue: 'defaultValue' },
+    },
+    {
+      field: 'defaultValueInteger',
+      type: ['int'],
+      cellEditorParams: { defaultValue: 10 },
+    },
+    {
+      field: 'defaultValueBoolean',
+      type: ['bool'],
+      cellEditorParams: { defaultValue: true },
+    },
+    {
+      field: 'defaultValueEnum',
+      type: ['enum'],
+      cellEditorParams: { defaultValue: 'Third' },
+    },
+    {
+      field: 'defaultValueDate',
+      type: ['date'],
+      cellEditorParams: { defaultValue: '01/01/2000' },
+    },
+    {
+      field: 'defaultValueNumber',
+      type: ['number'],
+      cellEditorParams: { defaultValue: '3.14' },
+    },
+    // Use of min values
+    {
+      field: 'minValueInteger',
+      type: ['int'],
+      cellEditorParams: { minValue: '42' },
+    },
+    {
+      field: 'minValueDate',
+      type: ['date'],
+      cellEditorParams: { minValue: '01/01/2030' },
+    },
+    {
+      field: 'minValueNumber',
+      type: ['number'],
+      cellEditorParams: { minValue: '3.14' },
+    },
+    // Webapp use maxValues because webapps values is higher
+    {
+      field: 'maxValueInteger',
+      type: ['int'],
+      cellEditorParams: { maxValue: '-42' },
+    },
+    {
+      field: 'maxValueNumber',
+      type: ['number'],
+      cellEditorParams: { maxValue: '-3.14' },
+    },
```

Comment:
Could you add tests cases with no `minValue` defined and `maxValue` > 0? (IIRC, the default value must be 0 in this case)

Reply:
DONE